src/panels/ImageDropdownUI.tsx

```diff
@@ -19,10 +19,17 @@ export const ImageDropdownUI = observer(function ImageDropdownUI_(p: { img: Medi
 export const ImageDropdownMenuUI = observer(function ImageDropdownMenuUI_(p: { img: MediaImageL }) {
     const st = useSt()
     const img = p.img
-
     return (
         <>
             <div className='divider divider-start my-0'>send to</div>
+            <MenuItem
+                icon={<span class='material-symbols-outlined'>content_copy</span>}
+                onClick={async () => {
+                    img.copyToClipboard()
+                }}
```

Comment:
as a rule of thumbs, if possible I'd say it's better to avoid curly brackets here; it will propagate the value

improvement 1 :

```ts
onClick={async () => img.copyToClipboard()}
```

improvement 2 :

```ts
// no need for async, since we're not using await
onClick={() => img.copyToClipboard()}
```

improvement 3 :

```ts
// no need for an extra lambda (Just have to make sure the copyToClipboard function is defined in MediaImageL using the `copyToClipboard = () => ...` syntax, and not using the `copyToClipboard() { ... }` syntax, but that's the case)
onClick={img.copyToClipboard}
```